api/users/views.py

```diff
@@ -577,7 +596,7 @@ def get_queryset(self):
         return user.affiliated_institutions.all()
 
 
-class UserRegistrations(UserNodes):
+class UserRegistrations(JSONAPIBaseView, generics.ListAPIView, UserMixin, NodeODMFilterMixin, NodesListFilterMixin):
```

Comment:
`UserNodes` has been updated to handle all of the filtering necessary for the profile page (addition of the `NodesFilterMixin` class). I didn't want to change `UserRegistrations` in this PR too, so this class now inherits from what `UserNodes` used to.